<frozen importlib._bootstrap>:228: RuntimeWarning: scipy._lib.messagestream.MessageStream size changed, may indicate binary incompatibility. Expected 56 from C header, got 64 from PyObject
Traceback (most recent call last):
  File "/mnt/home/ashwinr/.conda/envs/btp-mcis/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/mnt/home/ashwinr/.conda/envs/btp-mcis/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/mnt/home/ashwinr/btp24/grph/ISONET/subgraph/iso_matching_models.py", line 436, in <module>
    seed = config['seed']
  File "/mnt/home/ashwinr/btp24/grph/ISONET/subgraph/iso_matching_models.py", line 214, in train
    #losses = torch.nn.functional.mse_loss(target, prediction,reduction="sum")
KeyboardInterrupt
